.env

```diff
@@ -47,3 +47,9 @@ POSTFIX_RELAYHOST=
 POSTFIX_RELAYHOST_PORT=587
 POSTFIX_SASL_AUTH=
 POSTFIX_TLS=
+
+# Configure docker log driver - currently supports 'none', 'syslog' or 'journald' or 'json-file' (assuming no tags)
+LOG_DRIVER=json-file
+
+# Configure the moodle backup path
+MOODLE_BACKUP_ROOT=./backup
```

Comment:
The backup container would place DB backups in this directory? Just wondering how it works.